lib/core/storage.js

```diff
@@ -19,71 +19,23 @@ var _ = require('lodash');
  * Create our the store we will be using
  *
  * @param {Object}  options
- * @param {Boolean} options.isChromeApp
  * @param {Object}  options.ourStore the storage system we are using
```

Comment:
The `ourStore` is always localStorage in the uploader now, but as far as this wrapper goes, it does provide a little more than `localStorage` does. This relationship could be cleaned up/compressed, but I'm pretty sure that other moving pieces of the application (including `platform-client`) expect the API expressed by this wrapper to be available.